spacify-engine/src/main/java/org/uci/spacifyEngine/services/AvailableSlotsService.java

```diff
@@ -0,0 +1,80 @@
+package org.uci.spacifyEngine.services;
+
+import com.fasterxml.jackson.databind.util.JSONPObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.uci.spacifyLib.entity.AvailableSlotsEntity;
+import org.uci.spacifyLib.entity.ReservationEntity;
+import org.uci.spacifyLib.entity.RoomEntity;
+import org.uci.spacifyLib.entity.RoomType;
+import org.uci.spacifyLib.repsitory.AvailableSlotsRepository;
+import org.uci.spacifyLib.repsitory.ReservationRepository;
+import org.uci.spacifyLib.repsitory.RoomRepository;
+
+import java.math.BigInteger;
+import java.sql.Time;
+import java.time.LocalDateTime;
+import java.util.*;
+
+@Service
+public class AvailableSlotsService {
+
+    @Autowired
+    private RoomRepository roomRepository;
+
+    @Autowired
+    private ReservationRepository reservationRepository;
+
+    @Autowired
+    private AvailableSlotsRepository availableSlotsRepository;
+
+    public List<ReservationEntity> getAllAvailableSlots() {
+        String room_type = "STUDY";                                                     //to take from UI
+        LocalDateTime from_date_time = LocalDateTime.parse("2023-02-20T13:00");     //to take from UI
+        LocalDateTime to_date_time = LocalDateTime.parse("2023-02-20T15:00");       //to take from UI
+
+        List<BigInteger> room_ID = new ArrayList<>();
+        List<AvailableSlotsEntity> AvailableSlotsEntityList;
```

Comment:
can we rename to availableSlotsEntityList instead? Lower case for variables makes the code more readable. Like when I read line 40, I was confused, I thought you were doing get method call on a class